calico_kubernetes/calico_kubernetes.py

```diff
@@ -2,12 +2,16 @@
 import json
 import os
 import sys
+import docker
+from docker import Client
 from subprocess import check_output, CalledProcessError, check_call
 import requests
 import sh
 
 # Append to existing env, to avoid losing PATH etc.
 # Need to edit the path here since calicoctl loads client on import.
+DOCKER_VERSION = "1.16"
```

Comment:
I don't think this is the right place for this, just because the comment above doesn't apply to this line.

Could you move `DOCKER_VERSION=...` away from that comment?